<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgSketchOptions</class>
 <widget class="QWidget" name="WdgSketchOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>210</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item column="0" row="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Line width:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item column="1" row="0">
       <widget class="KisDoubleSliderSpinBox" name="lineWidthSPBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item column="0" row="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Offset scale:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item column="1" row="1">
       <widget class="KisDoubleSliderSpinBox" name="offsetSPBox"/>
      </item>
      <item column="0" row="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Density:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item column="1" row="2">
       <widget class="KisDoubleSliderSpinBox" name="densitySPBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>406</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item column="0" row="0">
       <widget class="QCheckBox" name="distanceDensityCHBox">
        <property name="text">
         <string>Use distance density</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item column="1" row="0">
       <widget class="QCheckBox" name="simpleModeCHBox">
        <property name="text">
         <string>Simple mode</string>
        </property>
       </widget>
      </item>
      <item column="0" row="1">
       <widget class="QCheckBox" name="magnetifyCHBox">
        <property name="text">
         <string>Magnetify</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item column="1" row="1">
       <widget class="QCheckBox" name="connectionCHBox">
        <property name="text">
         <string>Paint connection line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="randomRGBCHbox">
        <property name="text">
         <string>Random RGB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="randomOpacityCHbox">
        <property name="text">
         <string>Random opacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="distanceOpacityCHbox">
        <property name="text">
         <string>Distance opacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>